src/glue/v2/communication.hpp

```diff
@@ -35,7 +35,7 @@ namespace memgraph::glue::v2 {
 /// @param storage::v3::View for deciding which vertex attributes are visible.
 ///
 /// @throw std::bad_alloc
-storage::v3::Result<communication::bolt::Vertex> ToBoltVertex(
+storage::v3::ShardResult<communication::bolt::Vertex> ToBoltVertex(
```

Comment:
Does it make sense to still use shard result here? The data pumped into these functions are already resigning on the query engine, thus the shard result doesn't make sense in my opinion here. I know it is something that you haven't modified technically, but I vote for removing things instead of carrying over a flaw.